recipes-samples/images/rpb-console-image.bb

```diff
@@ -13,6 +13,9 @@ CORE_IMAGE_BASE_INSTALL += " \
     packagegroup-rpb \
 "
 
+# docker pulls runc/containerd, which in turn recommend lxc unecessarily
+BAD_RECOMMENDATIONS_append = " lxc"
```

Comment:
That's the one I'm currently working on and is close to my minimal image. So I'm focusing on this image to avoid to introduce the minimal image (they are pretty close).

Graphics images are more cluttered for various other reasons. I'll take a closer look to them when time permit.